<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buyTicketDialog</class>
 <widget class="QDialog" name="buyTicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: none; background-color: transparent;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(240, 252, 255);
border-radius:8px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="priceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rejectButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#rejectButton{border-radius:6px;
background-color:/*#44cef6;*/
rgb(248,248,248);

color:black}
QPushButton#rejectButton:hover{background-color:#e9e7ef;}
QPushButton#rejectButton:pressed{background-color:#c2ccd0}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acceptButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#acceptButton{border-radius:6px;
background-color:/*#44cef6;*/
#2ca9e1;
color:#ffffff}
QPushButton#acceptButton:hover{background-color:#44cef6;}
QPushButton#acceptButton:pressed{background-color:#007bbb}</string>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(240, 252, 255);</string>
    </property>
    <property name="text">
     <string>请确认你的订单：</string>
    </property>
   </widget>
   <widget class="QLabel" name="airlineLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>360</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img_background/mainbackground.png);
border-radius:0;
border-top-right-radius:8px;
border-top-left-radius:8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>193</y>
      <width>400</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2ca9e1</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>userLabel</zorder>
   <zorder>priceLabel</zorder>
   <zorder>rejectButton</zorder>
   <zorder>acceptButton</zorder>
   <zorder>airlineLabel</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>line</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rejectButton</sender>
   <signal>clicked()</signal>
   <receiver>buyTicketDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>buyTicketDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
